components/brave_rewards/browser/rewards_protocol_navigation_throttle.cc

```diff
@@ -103,24 +140,39 @@ void LoadRewardsURL(const GURL& redirect_url,
 
   if (IsValidWalletProviderRedirect(web_contents->GetURL(), redirect_url,
                                     kAllowedReferrerUrls)) {
-    web_contents->GetController().LoadURL(
-        TransformUrl(redirect_url), content::Referrer(), page_transition, "");
+    base::SequencedTaskRunner::GetCurrentDefault()->PostTask(
+        FROM_HERE, base::BindOnce(
+                       [](base::WeakPtr<WebContents> web_contents,
+                          const GURL& redirect_url) {
+                         if (!web_contents) {
+                           return;
+                         }
+                         web_contents->GetController().LoadURL(
+                             TransformUrl(redirect_url), content::Referrer(),
+                             ui::PAGE_TRANSITION_AUTO_TOPLEVEL, "");
```

Comment:
Just double-checking: is `PAGE_TRANSITION_AUTO_TOPLEVEL` the right `PageTransition` enumerator to use here?

Reply:
it was a suggestion from @bridiver